# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/thunderbird/hotfix-icu4c.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- thunderbird-144.0.1/intl/lwbrk/LineBreaker.cpp.vanilla	2025-11-10 12:17:40.675000000 +0100
+++ thunderbird-144.0.1/intl/lwbrk/LineBreaker.cpp	2025-11-10 12:18:30.329544793 +0100
@@ -454,6 +454,7 @@
       /* AKSARA_START = 45,                 [AS] */ CLASS_CHARACTER,
       /* VIRAMA_FINAL = 46,                 [VF] */ CLASS_CHARACTER,
       /* VIRAMA = 47,                       [VI] */ CLASS_CHARACTER,
+      /* U_LB_UNAMBIGUOUS_HYPHEN = 48,      [HH] */ CLASS_CHARACTER,
   };
 
   static_assert(U_LB_COUNT == std::size(sUnicodeLineBreakToClass),
